appveyor.yml

```diff
@@ -22,7 +22,7 @@ install:
   - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
 
   # Install the dependencies of the project (skip nibabel for speed)
-  - "conda install --yes --quiet setuptools numpy scipy matplotlib scikit-learn nose mayavi pandas h5py PIL patsy"
+  - "conda install --yes --quiet setuptools numpy scipy matplotlib scikit-learn nose mayavi pandas h5py PIL patsy pyside"
```

Comment:
When the GUIs are invoked the intended way through bash then we do set the proper backend: https://github.com/mne-tools/mne-python/blob/master/mne/commands/mne_kit2fiff.py#L47 

I can look into a way to detect the backend and raise an error if it's not qt (but maybe in a different PR?)

> and also skip the tests

You prefer to skip the tests? @Eric89GXL thought it was preferable to install the required backend